frontend/src/components/ScheduleCreation.vue

```diff
@@ -872,10 +873,6 @@ watch(
   display: block;
 }
 
-.bubble-select {
-  gap: .25rem;
-}
-
```

Comment:
This is handled by the BubbleSelect component itself now and hence not needed anymore.